.github/workflows/ci.yaml

```diff
@@ -0,0 +1,28 @@
+name: ci-pipeline
```

Comment:
can we rename this file to something similar to security-scan.yaml? Similarly, the name of the pipeline something like "security-checks".

Reply:
Yes this make sense for now.. but I was thinking for long term.

File: ci.yaml 
Workflow name: ci-pipeline
Jobs: tfsec-scan, tf-docs, tf-lint, tf fmt (we can have all CI related jobs under this workflow only) 


@KonradSchieban let me know if that makes sense